ubuntu_image/helpers.py

```diff
@@ -117,7 +117,7 @@ def snap(model_assertion, root_dir, channel=None, extra_snaps=None):
 
 
 def sparse_copy(src, dst, *, follow_symlinks=True):
-    args = ['cp', '-p', '--sparse=always', src, dst]
+    args = ['cp', '--preserve=mode,ownership', '--sparse=always', src, dst]
```

Comment:
As mentioned over in [this comment)[https://bugs.launchpad.net/ubuntu-image/+bug/1637554/comments/6], the owner, mode, and timestamps will be subtly different, at least across a remote file system (I can only test afp).  The copied file will have the uid of the remote user, but the mode doesn't match the remote user's umask.

Over a local file system then yes, it's the same process environment writing the original image file and sparse copying it, so I don't see how the mode or ownership could be different (the timestamp will be, but again, who cares?)

The corner case is if you don't run the state machine to completion, and then --resume it later, and in between you tweaked your environment, but I don't think that's a use case we need to sweat over. ;)

So I'm +1 on just removing -p